### ISSUE TITLE
bug: Filter with "{}" operator error

### ISSUE THREAD

**Issue by blueiceprj** [Netflix/dgs-framework#625]
I use 4.7.3 version of dgs and my query is running without any problem. But after update to 4.7.4 version I get this error. My query is:

```
{
  students(filter: {}) {
    status
    currentGradeLevel
    gradeLevel {
      shortName
      order
    }
    profile {
      firstName
      dateOfBirth
    }
  }
}
```

## Expected behavior

Query run

## Actual behavior
Error Message

```
com.netflix.graphql.dgs.exceptions.DgsInvalidInputArgumentException: Input argument type 'class x.xxxx.xxxxxx.service.analysis.documents.FilterInput' doesn't match input {}
	at com.netflix.graphql.dgs.internal.InputObjectMapper.mapToJavaObject(InputObjectMapper.kt:109)
	at com.netflix.graphql.dgs.internal.DataFetcherInvoker.convertValue(DataFetcherInvoker.kt:240)
	at com.netflix.graphql.dgs.internal.DataFetcherInvoker.processInputArgument(DataFetcherInvoker.kt:207)
	at com.netflix.graphql.dgs.internal.DataFetcherInvoker.invokeDataFetcher(DataFetcherInvoker.kt:62)
	at com.netflix.graphql.dgs.internal.DgsSchemaProvider.createBasicDataFetcher$lambda-19(DgsSchemaProvider.kt:310)
	at com.netflix.graphql.dgs.internal.DgsSchemaProvider$$Lambda$670/0x0000000000000000.get(Unknown Source)
	at graphql.execution.ExecutionStrategy.fetchField(ExecutionStrategy.java:270)
	at graphql.execution.ExecutionStrategy.resolveFieldWithInfo(ExecutionStrategy.java:203)
	at graphql.execution.AsyncExecutionStrategy.execute(AsyncExecutionStrategy.java:60)
	at graphql.execution.Execution.executeOperation(Execution.java:165)
	at graphql.execution.Execution.execute(Execution.java:104)
	at graphql.GraphQL.execute(GraphQL.java:557)
	at graphql.GraphQL.parseValidateAndExecute(GraphQL.java:482)
	at graphql.GraphQL.executeAsync(GraphQL.java:446)
```
## Steps to reproduce

_Note: A test case would be highly appreciated, but we understand that's not always possible_

**Comment by berngp** [comment]
Could you please update to the latest version, 4.7.6?

Could you also share the method signature of your DataFetcher that has the InputArgument that fails?

**Comment by blueiceprj** [comment]
Nothing changed.

```
	@DgsData(parentType = "Query", field = "students")
	public List<Student> students(@InputArgument("filter") FilterInput filter)
	{
		if (filter == null)
		{
			return Collections.emptyList();
		}

		List<Student> result = repository.findAll(filter);
		return result;
	}

```

```
public class FilterInput
{
	private FilterField expression;
	private List<FilterField> and;
	private List<FilterField> or;


```

**Comment by berngp** [comment]
Thanks @blueiceprj , this is fixed in v4.8.0.
